projects/media-viewer/src/lib/viewers/pdf-viewer/annotations-view.injector.ts

```diff
@@ -14,7 +14,6 @@ export class AnnotationsViewInjector {
       this.viewContainerRef.clear();
       const pdfViewerHtml = pdfViewer.nativeElement as HTMLElement;
       let pages = pdfViewerHtml.querySelectorAll(".page");
-      const selectedAnnotationEvent = new Subject<string>();
 
       if (pages.length && pages.length > 0) {
         annotations.forEach(annotation => {
```

Comment:
Thought I asked to get rid of this .forEach

Reply:
is there a reason to use a for-loop instead of a for-each

Reply:
I guess it's just a matter of style, for me `for (const annotation of annotations)` is more natural as it uses language constructs rather than higher order functions. It performs better as well, but I guess that doesn't matter

Reply:
ok, happy to change it

Reply:
done